src/components/containers/SubplotAccordion.js

```diff
@@ -113,25 +114,28 @@ class SubplotAccordion extends Component {
       }
     });
 
-    let pieCounter = 0;
-    let tableCounter = 0;
+    const counter = {
+      pie: 0,
+      table: 0,
+      sunburst: 0,
+    };
+
     data.forEach((d, i) => {
-      if ((d.type === 'pie' && d.values) || d.type === 'table') {
+      if ((d.type === 'pie' && d.values) || d.type === 'table' || d.type === 'sunburst') {
         if (d.type === 'pie') {
-          pieCounter++;
+          counter.pie++;
         } else if (d.type === 'table') {
-          tableCounter++;
+          counter.table++;
+        } else if (d.type === 'sunburst') {
+          counter.sunburst++;
         }
+
         subplotFolds[i] = (
           <TraceFold
             key={i}
             traceIndexes={[i]}
             canDelete={false}
-            name={
-              d.type === 'pie'
-                ? `${_('Pie')} ${pieCounter > 1 ? pieCounter : ''}`
-                : `${_('Table')} ${tableCounter > 1 ? tableCounter : ''}`
-            }
+            name={capitalize(d.type) + ' ' + counter[d.type]}
```

Comment:
You left out the `Counter > 1` condition, you still need it...

Reply:
i think its ok, if the counter is at zero, that trace does not even exist in data, so there won't be an accordion for it..

unless im missing something else?

Reply:
ah right nevermind, it can be 1

Reply:
The condition says that if you only have 1 trace, don't include number in the subplot fold name. So you don't get "Pie 1", it'll be just "Pie". Numbers kick in only if you have more than 1. So you do need that condition, otherwise it's not pretty.